Autocoders/Python/src/fprime_ac/generators/templates/enums/enum_cpp.tmpl

```diff
@@ -52,6 +52,34 @@ namespace ${namespace} {
     return *this;
   }
 
+  ${name}& ${name} :: operator=(const t a)
+  {
+#for $item_name,$item_value,$item_comment in $items_list:
+#if not $item_value == ""
+#set $manual_values = True
+#else
+#set $manual_values = False
```

Comment:
This code doesn't right. It looks copy-pasted from code that checks whether a raw number is a valid enum value. But here we already have a `t` value, so we don't need this checking.

Reply:
I'm pretty sure we should delete lines 57-78 here.